ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -335,14 +339,19 @@ azure_file_system_terminate(WT_FILE_SYSTEM *file_system, WT_SESSION *session)
     return 0;
 }
 
+// Check if the object (file) exists in the Azure storage source.
 static int
 azure_file_exists(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, bool *existp)
 {
-    WT_UNUSED(file_system);
+    azure_file_system *azure_fs = reinterpret_cast<azure_file_system *>(file_system);
+    int ret;
+    size_t size = 0;
```

Comment:
np: new line after declaration. Please follow this format for the position of WT_UNUSED for all of the other functions as well

Reply:
fixed :)